pints/toy/_simple_egg_box.py

```diff
@@ -135,3 +156,17 @@ def sample(self, n):
         np.random.shuffle(x)
         return x
 
+    def distance(self, samples):
```

Comment:
Why this alias?

Reply:
@MichaelClerx This is an alias that is used in all toy methods and provides Martin with a uniform port of call for all methods to do mass performance testing. This is so that he/someone else needn't specify somewhere Rosenbrock -> distance whereas, Simple eggbox -> kl_divergence.